.travis.yml

```diff
@@ -16,7 +16,9 @@ before_install:
   - ./.travis/download_solc.sh
   - pip install codecov
 
-install: pip install -r requirements-dev.txt
+install:
+  - pip install -r requirements-dev.txt
+  - python setup.py develop
```

Comment:
When using pip it might be more consistent to use pip for installing the PFS too with
```sh
pip install -e .
```
I'm not sure it that makes any difference in the result..

Reply:
Good point, updated it.